snippets/ptp-PtpConfigGmWpc.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: grandmaster
+  namespace: openshift-ptp
+spec:
+  profile:
+  - name: "grandmaster"
+    ptp4lOpts: "-2 --summary_interval -4"
```

Comment:
I believe summary-interval-4 is not required for GM .since ptp4l is not correcting the offset . it could be left at default -1

Reply:
@josephdrichard ^^

Reply:
`ptp4lOpts: "-2 --summary_interval -4"` is in the zpt-site-generate container, `PtpConfigGmWpc.yaml`